components/brave_rewards/resources/ui/components/tableDonation/style.ts

```diff
@@ -17,7 +17,7 @@ export const StyledRemove = styled<{}, 'button'>('button')`
   display: inline-block;
 `
 
-export const StyledRemoveIcon = styled<{}, 'span'>('span')`
+export const StyledRemoveIcon = styled('span')<{}>`
```

Comment:
Dang- having to edit all of these is a PITA. [Nice job on the regex though](https://regex101.com/r/5X75t3/1/) (and cool site too, how it explains the matching)

Reply:
yes, this PR has just been one exploding PITA after another, but we had to get there eventually.